src/Form/Themes.php

```diff
@@ -26,11 +26,39 @@ class Themes {
 
 
 	/**
-	 * Get Registered themes
+	 * Themes constructor.
+	 */
+	public function __construct() {
+		$this->load();
```

Comment:
No side-effects in the constructor. The act of constructing an object should _only_ construct that object. If an object has some initialization, then make that a method to be called: `$themes->loadThemes()`, and call that after instantiation:
```php
$themes = new Themes();
$themes->loadThemes();
```
You can do that when `Give` instantiates it.

Reply:
https://github.com/impress-org/givewp/compare/7c2a09061baa...c96a437a6320